src/getting-started.md

```diff
@@ -4,555 +4,208 @@ pageHeading: Getting Started
-[2023-03-28 14:18:13] DEBUG: hash <= target
-Hash:   000001b0a8b6be5c01c2e79656080eea652a30f49b5d04565ad923753dd9c811
-Target: 00000359fca60000000000000000000000000000000000000000000000000000
-[2023-03-28 14:18:13] > {"method": "mining.submit", "params": ["", "1", "0000000000000000", "6422d610", "1b4f00da"], "id":4}
-[2023-03-28 14:18:13] < {"id":4,"error":null,"result":true}
-[2023-03-28 14:18:13] accepted: 1/1 (100.00%), 171.09 khash/s (yay!!!)
-```
-Eventually, the Translation Proxy log output will show sucessful share, which means you've run the configuration successfully!
-
-```log
-2023-03-28T11:08:51.052292Z  INFO translator_sv2::downstream_sv1::downstream: PROXY SERVER - ACCEPTING FROM DOWNSTREAM: 127.0.0.1:50225
-2023-03-28T11:08:51.052704Z  INFO translator_sv2::downstream_sv1::downstream: Down: Subscribing
-2023-03-28T11:08:51.054303Z  INFO translator_sv2::downstream_sv1::downstream: Down: Authorizing
-2023-03-28T11:08:54.552777Z  INFO roles_logic_sv2::utils: reduce_path: coinbase_id: [55, 0, 43, 216, 197, 11, 114, 249, 81, 1, 236, 139, 193, 183, 192, 161, 226, 175, 179, 206, 208, 147, 206, 20, 71, 31, 50, 216, 79, 177, 68, 0]
-2023-03-28T11:08:54.553161Z  INFO translator_sv2::proxy::bridge: SHARE MEETS DOWNSTREAM TARGET
-```
-> <ins>**Warning**</ins> <br>
-> If Translation Proxy log outputs something like:
-> 
-```log
-2023-03-28T12:19:56.605826Z ERROR translator_sv2::proxy::bridge: Submit share error Ok("difficulty-too-low")
-2023-03-28T12:19:56.606004Z ERROR translator_sv2::proxy::bridge: Make sure to set `min_individual_miner_hashrate` in the config file
-```
-> You have to stop the tProxy execution, edit the difficulty parameters in your `proxy-config.toml`, to better adjust them accordingly to your miner, and run it again!
+
+On the CPUMiner directory:
+ 
+`./minerd -a sha256d -o stratum+tcp://localhost:34255 -q -D -P`
+## III Final Step: monitoring for blocks
+
+Once set up, monitor the machines and role logs for any valid blocks found.
+
+Following pictures detail the logs from tProxy, JDC, and Pool when a block is found.
+
+Pool logs
+
+![SRI Pool](/assets/SV2Pool-logs.png)
+
+Job Declarator Client (JDC) logs
+
+![SRI JDC](/assets/JobDeclaratorClient-JDC-logs.png)
+
+Translation Proxy tProxy logs
+
+![SRI tProxy](/assets/TranslationProxy-tProxy-logs.png) 
+
+### Monitor for mined blocks on Testnet
+
+If you didn’t change anything in the configuration files present in every role subdirectory, the valid block mined will be recognized by the string `Stratum v2 SRI Pool (0x5374726174756d2076322053524920506f6f6c)` inserted into the scriptSig of the coinbase tx input. 
+
+A way to look for it is to check the address `tb1qa0sm0hxzj0x25rh8gw5xlzwlsfvvyz8u96w3p8` since the default coinbase output is configured to be a P2WPKH built with a public key = `036adc3bdf21e6f9a0f0fb0066bf517e5b7909ed1563d6958a10993849a7554075`
+
+You can performa search on one of the following block explorers:
+- https://mempool.space/testnet 
```

Comment:
Hypertext is not working.
Maybe it is intended?

Reply:
https://github.com/stratum-mining/stratumprotocol.org/pull/191/commits/f7a1ab5edd1a1deb3f926f6fcc898398a353d056